README.md

```diff
@@ -1,58 +1,106 @@
+- Caching using the .NET `MemoryCache` to:
+  - Minimize round trips (pluggable via [`ICache`](https://github.com/jet/equinox/blob/master/src/Equinox.Core/Cache.fs#L22) thanks to [@DSilence](https://github.com/jet/equinox/pull/161)
+  - Minimize latency and bandwidth / Request Charges by maintaining the folded state, without making the Domain Model folded state serializable
+- Mature and comprehensive logging (using [Serilog](https://github.com/serilog/serilog) internally), with optimal performance and pluggable integration with your apps hosting context (we ourselves typically feed log info to Splunk and the metrics embedded in the `Serilog.Events.LogEvent` Properties to Prometheus; see relevant tests for examples)
+- **`Equinox.EventStore` In-stream Rolling Snapshots**: 
+  - No additional round trips to the store needed at either the Load or Sync points in the flow
+  - Support for multiple co-existing compaction schemas for a given stream (A 'compaction' event/snapshot is an Event). This is done by the [`FsCodec.IEventCodec`](https://github.com/jet/FsCodec#IEventCodec)
+    - Compaction events typically do not get deleted (consistent with how EventStore works), although it is safe to do so in concept
+  - While snapshotting can deliver excellent performance especially when allied with the Cache, [it's not a panacea, as noted in this excellent EventStore.org article on the topic](https://eventstore.org/docs/event-sourcing-basics/rolling-snapshots/index.html)
+- **`Equinox.CosmosStore` 'Tip with Unfolds' schema**: 
+  - In contrast to `Equinox.EventStore`'s `AccessStrategy.RollingSnapshots`, when using `Equinox.CosmosStore`, optimized command processing is managed via the `Tip` - a document per stream with an identity enabling syncing the read/write position via a single point-read. The `Tip` has the following attributes:
+    - It represents the position of the stream - i.e. the index at which the next events will be appended for a given stream (events and the Tip share a common logical partition key)
+    - It is ephemeral (`deflate+base64` compressed) [_unfolds_](DOCUMENTATION.md#Cosmos-Storage-Model)
+    - It can also be a holding buffer for events. Note: [presently removed](https://github.com/jet/equinox/pull/58), but [should return](DOCUMENTATION.md#Roadmap), see [#109](https://github.com/jet/equinox/pull/109)
```

Comment:
see the updated text. It's not 'presently removed' anymore!

Reply:
I reckon b and c are pretty OK as they got a bit of a rewrite (mainly because they are actually implemented)